exercises/diffie-hellman/src/test/java/DiffieHellmanTest.java

```diff
@@ -0,0 +1,95 @@
+import org.junit.Before;
```

Comment:
Out of interest, did you make this test suite yourself?

I know there is no canonical data so far - did you think of these tests or port them from another exercise? 🙂

Reply:
I port them from C# track [DiffieHellmanTest.cs](https://github.com/exercism/csharp/blob/master/exercises/diffie-hellman/DiffieHellmanTest.cs)